### PR TITLE
Remove expolosion particle

### PR DESCRIPTION
**Pull request by super626**
1. remove expolosion particles
2. fix transparent object render bug
3. add Animate3D::setTransitionTime

### CODE REVIEW

**Comment by super626** [comment]
[ci rebuild]

**Comment by super626** [comment]
[ci rebuild]

File: cocos/3d/CCAnimate3D.h [hunk 1]
```diff
@@ -89,9 +89,12 @@ class CC_DLL Animate3D: public ActionInterval
     void setOriginInterval(float interval);
     float getOriginInterval() const {return _originInterval; }
     
-    /** animate transition time */
+    /** get animate transition time between 3d animations */
     static float getTransitionTime() { return _transTime; }
     
+    /** set animate transition time between 3d animations */
+    static void setTransitionTime(float transTime) { if (transTime > 0.f) _transTime = transTime; }
```
Comment (darkdukey) [inline cocos/3d/CCAnimate3D.h]: Why can't set transition time to 0?

Comment (super626) [inline cocos/3d/CCAnimate3D.h]: Good. We can set it to zero. Zero means finishing transition at once.

**Comment by zilongshanren** [comment]
[ci rebuild]

**Pull request merged**